staging/src/k8s.io/client-go/pkg/api/helpers.go

```diff
@@ -465,6 +465,44 @@ const (
 	ObjectTTLAnnotationKey string = "node.alpha.kubernetes.io/ttl"
 )
 
+// AddOrUpdateTolerationInPod tries to add a toleration to the pod's toleration list.
+// Returns true if something was updated, false otherwise.
+func AddOrUpdateTolerationInPod(pod *Pod, toleration *Toleration) (bool, error) {
```

Comment:
This admission controller isn't supposed to reject pods. It's supposed to add the tolerations if they do not already exist, and otherwise do nothing. So I think the current behavior is correct?